package.json

```diff
@@ -14,26 +14,32 @@
   "dependencies": {
     "@jsdevtools/rehype-toc": "^3.0.2",
     "@svgr/webpack": "^6.5.0",
+    "@typescript-eslint/parser": "^7.7.1",
     "classnames": "^2.3.2",
     "framer-motion": "^7.6.7",
     "gray-matter": "^4.0.3",
     "image-size": "^1.0.2",
     "jwt-decode": "^3.1.2",
     "lodash.clonedeep": "^4.5.0",
     "minimatch": "^5.1.2",
-    "next": "13.0.0",
-    "next-image-export-optimizer": "^1.0.1",
+    "next": "^14.2.3",
+    "next-image-export-optimizer": "^1.13.0",
     "next-mdx-remote": "^4.1.0",
     "next-remote-watch": "^2.0.0",
+    "next-transpile-modules": "^10.0.1",
     "prism-react-renderer": "^1.3.5",
     "prismjs": "^1.29.0",
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "rehype-autolink-headings": "^6.1.1",
     "rehype-slug": "^5.1.0",
     "remark-gfm": "^3.0.1",
+    "test-yorkie-v1": "^0.8.4",
+    "typescript-eslint": "^7.7.1",
     "unist-util-visit": "^4.1.1",
-    "yorkie-js-sdk": "^0.4.19"
+    "yorkie": "^2.0.0",
```

Comment:
This package appears to have been added by mistake. What do you think?

Reply:
Yes, This is mistake the npm I will remove it.